packages/base/style/symbologyDialog.css

```diff
@@ -128,13 +128,6 @@ select option {
   margin: auto;
 }
 
-.jp-gis-loading-spinner {
```

Comment:
This CSS was redundant to the props on the `<FontAwesomeIcon>` component. E.g. `<FontAwesomeIcon spin />` replaces the `animation` CSS attribute. Same for the `size` prop and `font-size`.